scenariorun/src/main/kotlin/com/cosmotech/scenariorun/azure/ScenarioRunServiceImpl.kt

```diff
@@ -102,9 +102,26 @@ internal class ScenarioRunServiceImpl(
       // TODO Only the owner or an admin should be able to perform this operation
       throw CsmAccessForbiddenException("You are not allowed to delete this Resource")
     }
-    this.deleteScenarioRunWithoutAccessEnforcement(scenarioRun)
+    deleteScenarioRunWithoutAccessEnforcement(scenarioRun)
+    deleteScenarioRunMetadata(
```

Comment:
This method call is not needed: ADX csmSimulation drop by extent already done in   deleteScenarioRunWithoutAccessEnforcement line 135